_posts/2020-02-23-kfserving-0.5.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Operationalize, Scale and Infuse Trust in AI Models using KFServing"
+description: "KFServing 0.5 Blog Post"
+layout: post
+toc: false
+comments: true
+image: images/logo.png
+hide: false
+categories: [release, official]
+---
+
+#### By **Animesh Singh** and **Dan Sun**
+
+With inputs from : Yuzhui Liu, Tommy Li, Paul Vaneck, Andrew Butler, Srinivasan Parthasarathy
+
```

Comment:
Can we put at the front that this is from the KFServing Working Group? Would love to get WG attribution in blog posts.

Reply:
done